### ISSUE TITLE
Cannot start

### ISSUE THREAD

**Issue by PhuocLinh** [Uepaa-AG/p2pkit-quickstart-ios#3]
I have updated the SDK to version 1.3.5 and when to run the app, it has a problem.
The Discovery State always return PPKPeer2PeerDiscoverySuspended on the delegate function p2pDiscoveryStateChanged. With version 1.3.0, after returning value PPKPeer2PeerDiscoverySuspended, it returns value PPKPeer2PeerDiscoveryRunning quickly

So that I cannot detect any Bluetooth iPhone around

**Comment by thupa** [comment]
Hi,
`PPKPeer2PeerDiscoverySuspended` usually means that Bluetooth is suspended.

Can you tel us more about the type of device and operating system version?

Thanks!

**Comment by PhuocLinh** [comment]
My device is iPhone 5s (9.3.1), iPod touch 5 (8.4.1) and iPhone 6s(9.3.2)

**Comment by thupa** [comment]
Hi,
Has the issue been resolved? if yes, please close it.
Thank you

**Comment by orpp94** [comment]
Hi! I have just started implementing p2pKit into my project, it looks like pretty interesting although I have just found the same issue as this guy. PPKController cannot get initialized, instead, it throws me the "suspended state" notification.

I sure have verified having my bluetooth on. I am afraid it might be something with the app key, because after a few tries I have tried to enable the PPKController with a random string key and it still shows me the same problem. Also, having a look at your youtube video "Get Started..." I have notified how long is the key you use, mine is an alphanumeric 32 elements long (pretty much shorter). 

Would appreciate some help, thanks guys
